jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -202,7 +202,7 @@ def test_continuum_surrounding_spectral_subset(specviz_helper, spectrum1d):
     plugin.width = 3
 
     # Values have not yet been validated
-    np.testing.assert_allclose(float(plugin.results[0]['result']), 2.153181e-13, atol=1e-15)
+    np.testing.assert_allclose(float(plugin.results[0]['result']), 1.908697e-13, atol=1e-15)
```

Comment:
I get it if the Subset limits being returned are now what user provided instead of closes data match, but why would the science _result_ change?

Reply:
Not sure, it looks like those values have never been validated by a scientist. Does @camipacifici, @PatrickOgle, or @orifox want to check that out?

Reply:
There isn't much science going on here. When you visualize the test case, what is happening with this patch actually makes more sense; the continuum should never eat into selected data. *However,* with this patch, the continua on both sides are not symmetrical w.r.t. distance from data; is that desirable? I think what you are seeing here is that the selection from your patch included more data points for the "background," hence decreasing the line flux measured.

Is you ask me, I think both are wrong in their own ways.

On `main`:

![Screenshot 2023-03-21 132648](https://user-images.githubusercontent.com/2090236/226694006-bd9f80a6-32d8-47f2-bbc3-bbeb2b9d9fa7.jpg)

With this patch:

![Screenshot 2023-03-21 133354](https://user-images.githubusercontent.com/2090236/226694009-1d82b9b2-28a9-45de-8a89-edc5058be48f.jpg)

Reply:
I see that line analysis uses `get_subsets_from_viewer` and I wonder if that is the root cause. I can create a ticket for this and work off of this PR if that is accessible.